pom.xml

```diff
@@ -88,7 +88,8 @@
         <!-- core lib dependency -->
         <bytebuddy.version>1.12.13</bytebuddy.version>
         <grpc.version>1.44.0</grpc.version>
-        <gson.version>2.8.6</gson.version>
+        <netty.version>4.1.79.Final</netty.version>
```

Comment:
Why do you set the netty version separately? Isn't bumping up grpc version working?